### ISSUE TITLE
TypeError: Cannot read property 'htmlAttributes' of undefined

### ISSUE THREAD

**Issue by fdaciuk** [jaredpalmer/after.js#117]
Hi! Firstly, thanks for the really great work on after.js! :clap: :clap: 

I'm having a problem when I try to run `yarn start`. Debugging the code, I get this error on `src/server.js` - I added a `console.log(error)` on line `25`, inside `catch` statement.

I used the following commands from README to install it:

```sh
yarn global add create-razzle-app
create-razzle-app --example with-afterjs myapp
cd myapp
yarn start
```

`create-razzle-app` version: 0.8.12

I just did the installation and ran the `yarn start` to get this error.

**Comment by fdaciuk** [comment]
Updating: It worked when I removed the `@jaredpalmer/after` package, and added it using the version `1.1.0`.

**Comment by fredrik-sogaard** [comment]
Yes, it seems like 1.2.1 is broken. See #112 and #113

**Comment by fdaciuk** [comment]
Nice! Thanks @fredrik-sogaard =)

**Comment by John0x** [comment]
There seems to be a lot of stuff broken with 1.2.1

I've fixed the htmlAttributes problem locally by overwriting the document in the render function, but there are still other problems with the server side rendering. The server is rendering my components as "undefined". Client side is working though.

**Comment by jaredpalmer** [comment]
This has been fixed in 1.3.
